jujugui/static/gui/src/app/assets/css/base.scss

```diff
@@ -137,12 +137,18 @@ body {
     -moz-osx-font-smoothing: grayscale;
 }
 
+%link,
 .link,
-a,
-a:hover {
+a {
     cursor: pointer;
     color: $ubuntu-orange;
     text-decoration: none;
+
+    &:hover,
+    &:focus,
+    &:active {
+        text-decoration: underline;
```

Comment:
This is effecting the hover styled for the breadcrumb. You will need to remove the underline when hovering over the username.